Config/development/server.json

```diff
@@ -0,0 +1,3 @@
+{
+  "app-token": "test"
```

Comment:
Why app? Let's use client. It's more correctly cuz we wanted to give data for our site through api.

Reply:
Fixed